.github/ISSUE_TEMPLATE/timebox.md

```diff
@@ -0,0 +1,23 @@
+---
+name: ⏳ Timebox
+about: Specify an effort that will be completed within a pre-defined amount of time.
+title: ''
+labels: 'timebox'
+assignees: ''
+
+---
+
+## Related user story
+
+TODO
+<!-- A timebox always belongs to exactly one story.  The parent user story for this timebox is linked here.   Comment on the parent story, not on this sub-task. -->
+
+## Task
+
+TODO
+<!-- What needs to be done. -->
+
+## Condition of satisfaction
+
+TODO
+<!-- Describe the conditions of satisfaction that will resolve this issue.  The "definition of done". It is always up to contributors to check their own work.  But especially keep in mind there is no external quality assurance check for timeboxes.  (Only user stories get automatic external QA.  With timeboxes, it's up to you.)  -->
```

Comment:
delete ~~But especially keep in mind there is no external quality assurance check for timeboxes.  (Only user stories get automatic external QA.  With timeboxes, it's up to you.)~~